File: www/scripts/parser.php
<?php 
    echo "Parseo de variables: ";
    echo $argv[1];

    $variables = array();

    $args = explode("&", $argv[1]);
    foreach($args as $arg)
    {
        $vals = explode("=", $arg);
        $variables[$vals[0]] = urldecode($vals[1]);
    }

    print_r($variables);
    print("\r\n");

    print("Unreachable by server");
?>

File: www/scripts/parser.py
import sys
from urllib.parse import unquote

print("Parseo de variables: ")
variables = dict()

try:
    for arg in sys.argv[1].split("&"):
        name, value = arg.split("=")
        variables[name] = unquote(value)
except:
    print("ERROR: Contenido incorrecto", end='\r\n')
    
print(variables, end='\r\n')

print("Unreachable by server")